### ISSUE TITLE
no-unused-modules generates `parseForESLint is invalid and will just be ignored` 

### ISSUE THREAD

**Issue by sobolevn** [benmosher/eslint-plugin-import#1601]
Output: 

```
» npm run lint:js

> name@0.0.0 lint:js /Users/sobolev/Documents/github/wemake-vue-template/template
> eslint --cache --ext .js,.ts,.vue,.json --ignore-path .gitignore .

`parseForESLint` from parser `@typescript-eslint/parser` is invalid and will just be ignored
`parseForESLint` from parser `@typescript-eslint/parser` is invalid and will just be ignored
`parseForESLint` from parser `@typescript-eslint/parser` is invalid and will just be ignored
`parseForESLint` from parser `@typescript-eslint/parser` is invalid and will just be ignored
```

Full config: https://github.com/wemake-services/wemake-frontend-styleguide/blob/master/packages/eslint-config-typescript/base.js

Disabling this line helped to remove this warning: https://github.com/wemake-services/wemake-frontend-styleguide/blob/master/packages/eslint-config-typescript/base.js#L69-L71

Reproduction steps:

1. Clone this project, install deps, https://github.com/wemake-services/wemake-frontend-styleguide
2. Uncomment this line https://github.com/wemake-services/wemake-frontend-styleguide/blob/7ad6f579d2b2995efcce37e75c04d67f143403e3/packages/eslint-config-typescript/base.js#L71
3. Run `yarn lint:ts`

Original issue: https://github.com/typescript-eslint/typescript-eslint/issues/1333

**Comment by ljharb** [comment]
This warning may be coming from this plugin, but for the parser to be compatible with this package, it has to implement the right methods. This seems like a bug in the parser to me.

**Comment by sobolevn** [comment]
@ljharb parser authors said that it is on your side: https://github.com/typescript-eslint/typescript-eslint/issues/1333#issuecomment-566036856

Anything I can help with?

**Comment by ljharb** [comment]
Closing per https://github.com/typescript-eslint/typescript-eslint/issues/1333#issuecomment-573332830 ; happy to reopen if it turns out there’s a change needed here.

**Comment by Ranguna** [comment]
@ljharb  I'm having this problem now, not sure what I'm supposed to do.

**Comment by Ranguna** [comment]
Hmmm... It looks like the problem solved itself after a while ..?
Strange.
